### PR TITLE
209: Move button bindings

### PR DESCRIPTION
**Pull request by BrianLitwin**
addresses part 1 of #209 - ItemListView 
Connected to #209 

The updates in ItemListViewSpec unit test file mirror the way button taps are tested in WelcomeViewSpec.. I'm not 100% on those, probably want to double-check those in review.

### CODE REVIEW

File: lockbox-iosTests/ItemListViewSpec.swift [hunk 1]
```diff
@@ -287,16 +281,42 @@ class ItemListViewSpec: QuickSpec {
                     }
                 }
             }
-
+
```
Comment (sashei) [inline lockbox-iosTests/ItemListViewSpec.swift]: 👾

File: lockbox-ios/Presenter/ItemListPresenter.swift [hunk 2]
```diff
@@ -142,6 +110,44 @@ class ItemListPresenter: BaseItemListPresenter {
...
+                                    checked: latest == Setting.ItemListSort.recentlyUsed),
+                                AlertActionButtonConfiguration(
+                                    title: Constant.string.cancel,
+                                    tapObserver: nil,
+                                    style: .cancel)
+                            ],
+                            title: Constant.string.sortEntries,
+                            message: nil,
+                            style: .actionSheet)
+                    }).disposed(by: self.disposeBag)
```
Comment (sashei) [inline lockbox-ios/Presenter/ItemListPresenter.swift]: nit: put the `disposed` on a new line like the rest of the Observable operators.

File: lockbox-ios/Presenter/ItemListPresenter.swift [hunk 3]
```diff
@@ -40,47 +42,13 @@ class ItemListPresenter: BaseItemListPresenter {
         }.asObserver()
     }
```
Comment (sashei) [inline lockbox-ios/Presenter/ItemListPresenter.swift]: 😍

**Comment by BrianLitwin** [comment]
@sashei I'll fix up as per the comments.. Is there something I should do to address the failed buddybuild on the last commit? I haven't worked with buddybuild before. Thanks for the feedback!

**Comment by sashei** [comment]
@BrianLitwin There is an unrelated problem with the `uispecs` right now so don't worry about addressing that... I am rebuilding to see if it resolves the build failure, which looks unrelated to your code.

**Comment by BrianLitwin** [comment]
Requested changes are in the 6th commit!

**Pull request merged**